saver.go

```diff
@@ -17,13 +18,16 @@ type SaveQuery struct {
 
 type Saver interface {
 	Insert(structPtr interface{}) (sql.Result, error)
+	InsertWithContext(ctx context.Context, structPtr interface{}) (sql.Result, error)
 	QueryForInsert(structPtr interface{}) (*SaveQuery, error)
 	Update(table string, set map[string]interface{}, where Clause) (sql.Result, error)
+	UpdateWithContext(ctx context.Context, table string, set map[string]interface{}, where Clause) (sql.Result, error)
 	QueryForUpdate(table string, set map[string]interface{}, where Clause) (*SaveQuery, error)
 }
 
 type Executor interface {
 	Exec(query string, args ...interface{}) (sql.Result, error)
+	ExecContext(ctx context.Context, query string, args ...interface{}) (sql.Result, error)
```

Comment:
Please change this also.

Reply:
"Database/sql" doesn't have `ExecWithContext` , so I don't think can change it.

Reply:
@keroxp Please check this comment 🙏

Reply:
@TomoakiMizuno I got it! Would you revert previous rename changes into original version?

Reply:
@keroxp 33a35c8 This changes reverted.